app/assets/stylesheets/layout/_top_menu.sass

```diff
@@ -51,37 +51,51 @@
   %absolute-layout-mode &
     position: absolute
 
-  #search
-    margin: 5px 25px 0 0
+  #top-menu-search
     float: left
-  #search_wrap
+  .search-wrapper
     position: relative
-    top: 6px
+  .search_btn
+    display: block
+    height: $header-height
+    line-height: $header-height
+    @include default-font($header-item-font-color, $header-item-font-size)
+    &:hover
+      text-decoration: none
+      background: $header-item-bg-hover-color
+      color: $header-item-font-hover-color
   .search_field
-    border: $header-search-field-border
-    border-radius: 25px
-    width: 180px
-    outline: 0
-    padding: 7px 30px 7px 10px
-    font-size: 0.75rem
-    color: $header-search-field-font-color
-  ::-webkit-input-placeholder
-    color: $header-search-field-font-color
-  :-moz-placeholder
-    color: $header-search-field-font-color
-  span#magnifier
-    @include icon-common
-    position: absolute
-    top:   9px
-    right: 9px
-    &:after
+    display: none
```

Comment:
I actually liked the old rule (`display: none` for `.search-collapsed .search_field`) more. Because there is no apparent reason why a lonely `search_field` should not be displayed, for a collapsed search field this is more obvious.

Was there a technical reason for that change in https://github.com/oliverguenther/openproject/commit/3b2027b42cf3956e4803a9295347b5d8dab4cdb6?